daniel-aguirre-salamanca/project/src/App.js

```diff
@@ -0,0 +1,65 @@
+import "./App.css";
+import React, { useEffect } from "react";
+import { Route, Routes, useLocation } from "react-router-dom";
+import { Home, People, Planets, Movies, Vehicles} from "./views";
+import { Header, Footer, Loading } from "./components/static";
+import { useDispatch, useSelector } from "react-redux";
+import {addData} from "./features/dataSlice";
+
+function App() {
+  const status = useSelector(state=> state.users.status);
+  const isData = useSelector(state => state.data);
+  const location = useLocation();
+  const dispatch = useDispatch();
+
+  const getData = (type) => {
```

Comment:
@daguirreglobant all the logic and data consumption should be moved to a Context

Reply:
Really interesting the use of Thunk(redux) for removing the logic and data consumption from the component App.js, thank you!